saleor/tests/e2e/checkout/test_unlogged_customer_should_not_be_able_to_order_product_quantity_greater_than_stock.py

```diff
@@ -0,0 +1,127 @@
+import pytest
+
+from ..channel.utils import create_channel
+from ..product.utils import (
+    create_category,
+    create_product,
+    create_product_channel_listing,
+    create_product_type,
+    create_product_variant,
+    create_product_variant_channel_listing,
+)
+from ..shipping_zone.utils import create_shipping_zone
+from ..utils import assign_permissions
+from ..warehouse.utils import create_warehouse, update_warehouse
+from .utils import raw_checkout_create
+
+
+def prepare_product(
+    e2e_staff_api_client,
+    permission_manage_products,
+    permission_manage_channels,
+    permission_manage_product_types_and_attributes,
+    permission_manage_shipping,
+):
+    permissions = [
+        permission_manage_products,
+        permission_manage_channels,
+        permission_manage_product_types_and_attributes,
+        permission_manage_shipping,
+    ]
+
+    assign_permissions(e2e_staff_api_client, permissions)
+    warehouse_data = create_warehouse(e2e_staff_api_client)
+    warehouse_id = warehouse_data["id"]
+
+    update_warehouse(
```

Comment:
We shouldn't call this when we don't want to update any warehouse data.

Reply:
@fowczarek https://github.com/saleor/saleor/pull/13362/commits/6972adb2b0f5c9a0c3220bc1ed1ef139fca8fa90